### PR TITLE
docs: reorg integrate SDK

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

reorg integrate SDK docs and add `<Step>` component to render content properly

## Testing

image

### CODE REVIEW

File: packages/docs/docs/tutorial/integrate-sdk/React.mdx [hunk 1]
```diff
@@ -1,3 +1,8 @@
+---
+sidebar_label: React
+---
+
+import Step from '@site/src/components/Step';
```
Comment (demonzoo) [inline packages/docs/docs/tutorial/integrate-sdk/React.mdx]: Does it have to start with `@site/src`? Would it be possible to use `@/components`?

**Review by demonzoo — COMMENTED** [review]
The CI is failed

File: packages/docs/docs/tutorial/integrate-sdk/React.mdx (hunk 1 above)
Comment (gao-sun) [reply packages/docs/docs/tutorial/integrate-sdk/React.mdx]: or `@components/`?

**Review by demonzoo — APPROVED** [review]

**Comment by demonzoo** [comment]
Looks good but please check CI build failure

**Pull request merged**